### PR TITLE
Add gemfile

### PR DESCRIPTION
**Pull request by summera**

### CODE REVIEW

**Comment by summera** [comment]
Makes it a bit easier for those looking to contribute :smile:

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 pkg
-.ruby-version
\ No newline at end of file
+.ruby-version
+/Gemfile.lock
```
Comment (rothsa) [inline .gitignore]: :+1:

File: email_reply_parser.gemspec [hunk 2]
```diff
@@ -96,5 +96,8 @@ Gem::Specification.new do |s|
   ## Test files will be grabbed from the file list. Make sure the path glob
   ## matches what you actually use.
   s.test_files = s.files.select { |path| path =~ /^test\/.*_test\.rb/ }
+
```
Comment (rothsa) [inline email_reply_parser.gemspec]: There is a comment block for add_development_dependency above, so maybe move this up there for neatness sake  :)

**Comment by rothsa** [comment]
@technoweenie looks good to me.

File: email_reply_parser.gemspec (hunk 2 above)
Comment (summera) [inline email_reply_parser.gemspec]: Good call :ok_hand: . Missed it for some reason

**Comment by technoweenie** [comment]
Lol ruby.  It never needed one until they broke test unit.

**Comment by summera** [comment]
@technoweenie I think this PR is still necessary to resolve the issue.

**Comment by rickpastoor** [comment]
:+1: would be great if this could be merged!

**Comment by summera** [comment]
@rickpastoor @rothsa I updated and merged this into my fork, as it doesn't look like a lot is going on here

https://github.com/summera/email_reply_parser

**Comment by mattyoho** [comment]
Thanks for opening this PR, @summera. I'm sorry that it sat for so long with no interaction.

Given that the gem can be developed locally without using `bundler` I'm going to close this out, but please continue the conversation if there's significant disagreement with that. :bow:

**Pull request closed without merging**